README.md

```diff
@@ -1,76 +1,62 @@
+first blockchain to provide [autonomous smart contracts](https://docs.massa.net/en/latest/general-doc/autonomous-sc.html)
+and native [front-end hosting](https://docs.massa.net/en/latest/general-doc/decentralized-web.html).
 
-You can watch a short introduction video [here](https://www.youtube.com/watch?v=HbILgK1Wh-4).
+Here is a list of tools to easily build applications on the Massa blockchain:
 
-We are now releasing the **Massa testnet** in this GitHub repository,
-with its explorer available at <https://massa.net/testnet>.
+- [JS Client library](https://github.com/massalabs/massa-web3) to connect to the Massa blockchain from your applications.
+- [AssemblyScript](https://github.com/massalabs/massa-as-sdk) SDKs to write smart contracts.
+- [Examples of applications](https://github.com/massalabs/massa-sc-examples) built on Massa.
+- [Explorer](test.massa.net).
+- [Interactive API specification](https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/v2&uiSchema\[appBar\]\[ui:input\]=false&uiSchema\[appBar\]\[ui:inputPlaceholder\]=Enter+Massa+JSON-RPC+server+URL&uiSchema\[appBar\]\[ui:logoUrl\]=https://massa.net/favicons/favicon.ico&uiSchema\[appBar\]\[ui:splitView\]=false&uiSchema\[appBar\]\[ui:darkMode\]=false&uiSchema\[appBar\]\[ui:title\]=Massa&uiSchema\[appBar\]\[ui:examplesDropdown\]=false&uiSchema\[methods\]\[ui:defaultExpanded\]=false&uiSchema\[methods\]\[ui:methodPlugins\]=true&uiSchema\[params\]\[ui:defaultExpanded\]=false).
+- [Lots of documentation](docs.massa.net), from [web3 development](https://docs.massa.net/en/latest/web3-dev/smart-contracts.html)
+  to [Massa's architecture](https://docs.massa.net/en/latest/general-doc/architecture.html).
 
-## Massa API
-
-You can interact with Massa JSON-RPC API via the [interactive API specification](https://playground.open-rpc.org/?schemaUrl=https://test.massa.net/api/v2&uiSchema\[appBar\]\[ui:input\]=false&uiSchema\[appBar\]\[ui:inputPlaceholder\]=Enter+Massa+JSON-RPC+server+URL&uiSchema\[appBar\]\[ui:logoUrl\]=https://massa.net/favicons/favicon.ico&uiSchema\[appBar\]\[ui:splitView\]=false&uiSchema\[appBar\]\[ui:darkMode\]=false&uiSchema\[appBar\]\[ui:title\]=Massa&uiSchema\[appBar\]\[ui:examplesDropdown\]=false&uiSchema\[methods\]\[ui:defaultExpanded\]=false&uiSchema\[methods\]\[ui:methodPlugins\]=true&uiSchema\[params\]\[ui:defaultExpanded\]=false) which is the equivalent of SwaggerUI for OpenRPC specifications.
-
-## Testnet Incentives
+## Join the Testnet
 
 As decentralization is our core value, we would like to help you start
-and run a Massa node. Running a node during the testnet phase also helps
+and run a Massa node. You'll find a full tutorial on how to run a node on the testnet
+[here](https://docs.massa.net/en/latest/testnet/install.html#testnet-install).
+
+Running a node during the testnet phase also helps
 us find bugs and improve usability, so it will be rewarded with real
```

Comment:
This is an attempt to rephrase in a way that might improve the rate at which people join testnet.

With decentralization as a core value, we've gone to great lengths to lower the barrier of entry for community participation in our community testnet, and we [invite you to join in](https://docs.massa.net/en/latest/testnet/install.html#testnet-install), and register for the testnet participation reward program.

Your participation will help improve decentralization, usability, and network reliability leading up to, and following, the launch of mainnet.